src/orders/components/OrderChangeWarehouseDialog/OrderChangeWarehouseDialog.tsx

```diff
@@ -71,13 +80,14 @@ export const OrderChangeWarehouseDialog: React.FC<OrderChangeWarehouseDialogProp
   });
   const filteredWarehouses = mapEdgesToItems(warehousesOpts?.data?.search);
 
+  const selectedWarehouse = filteredWarehouses?.find(
+    getById(selectedWarehouseId ?? "")
+  );
+
   const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     setSelectedWarehouseId(e.target.value);
   };
   const handleSubmit = () => {
-    const selectedWarehouse = filteredWarehouses.find(
-      getById(selectedWarehouseId)
-    );
     onConfirm(selectedWarehouse);
```

Comment:
Will there be a case where `selectedWarehouse` is empty?

Reply:
Generally yes, i can think of two cases:
- There are no search results
- There are search results, but currently selected warehouse is not among them.

However, confirm button is disabled then.